<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="progressBar1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QPushButton" name="switch1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Switch Player</string>
    </property>
   </widget>
   <widget class="QPushButton" name="switch2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Switch Player</string>
    </property>
   </widget>
   <widget class="QPushButton" name="second">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>120 sec</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minute">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>5 min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="gameInfoText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Valitse peliaika ja paina Start!</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>386</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
